onnx/reference/ops/op_gelu.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) ONNX Project Contributors
```

Comment:
Since it is defined as a function, this file should not be needed. ReferenceEvaluator will inline the operator. Plus, it is not used right now because it is not imported in https://github.com/onnx/onnx/blob/main/onnx/reference/ops/_op_list.py. So removing it should not change anything.

Reply:
@xadupre I'm a bit confused... I only added this file since @gramalingam mentioned this in his review above. Is this file not required then?

Reply:
Sorry about the confusion, my mistake. Please ignore my suggestion. The function-definition makes the reference implementation unnecessary. (I was thinking about the test-case generation logic, which does require an implementation. But let's leave it as is.)